frontends/concrete-python/tests/execution/test_comparison.py

```diff
@@ -2,160 +2,175 @@
+                    fhe.ComparisonStrategy.TWO_TLU_BIGGER_CASTED_SMALLER_PROMOTED,
+                    fhe.ComparisonStrategy.THREE_TLU_BIGGER_CLIPPED_SMALLER_CASTED,
+                    fhe.ComparisonStrategy.TWO_TLU_BIGGER_CLIPPED_SMALLER_PROMOTED,
+                    fhe.ComparisonStrategy.CHUNKED,
+                ]
+            ),
+        ]
+    )
+
+# pylint: disable=redefined-outer-name
+
+
 @pytest.mark.parametrize(
-    "parameters",
-    [
-        {
-            "x": {"range": [0, 3], "status": "encrypted"},
-            "y": {"range": [0, 3], "status": "encrypted"},
-        },
-        {
-            "x": {"range": [0, 255], "status": "encrypted"},
-            "y": {"range": [0, 255], "status": "encrypted"},
-        },
-        {
-            "x": {"range": [-128, 127], "status": "encrypted"},
-            "y": {"range": [-128, 127], "status": "encrypted"},
-        },
-        {
-            "x": {"range": [-128, 127], "status": "encrypted"},
-            "y": {"range": [0, 255], "status": "encrypted"},
-        },
-        {
-            "x": {"range": [0, 255], "status": "encrypted"},
-            "y": {"range": [-128, 127], "status": "encrypted"},
-        },
-        {
-            "x": {"range": [-8, 7], "status": "encrypted"},
-            "y": {"range": [-8, 7], "status": "encrypted", "shape": (2,)},
-        },
-        {
-            "x": {"range": [-8, 7], "status": "encrypted", "shape": (2,)},
-            "y": {"range": [-8, 7], "status": "encrypted"},
-        },
-        {
-            "x": {"range": [-8, 7], "status": "encrypted", "shape": (2,)},
-            "y": {"range": [-8, 7], "status": "encrypted", "shape": (2,)},
-        },
-    ],
+    "operation,"
+    "lhs_bit_width,rhs_bit_width,"
+    "lhs_is_signed,rhs_is_signed,"
+    "lhs_shape,rhs_shape,"
+    "strategy",
+    cases,
 )
-def test_comparison(function, parameters, helpers):
+def test_comparison(
+    operation,
+    lhs_bit_width,
+    rhs_bit_width,
+    lhs_is_signed,
+    rhs_is_signed,
+    lhs_shape,
+    rhs_shape,
+    strategy,
+    helpers,
+):
     """
     Test comparison operations between encrypted integers.
     """
 
-    parameter_encryption_statuses = helpers.generate_encryption_statuses(parameters)
-    configuration = helpers.configuration()
+    name, function = operation
 
-    compiler = fhe.Compiler(function, parameter_encryption_statuses)
+    lhs_dtype = Integer(is_signed=lhs_is_signed, bit_width=lhs_bit_width)
+    rhs_dtype = Integer(is_signed=rhs_is_signed, bit_width=rhs_bit_width)
 
-    inputset = helpers.generate_inputset(parameters)
-    circuit = compiler.compile(inputset, configuration)
+    lhs_description = ValueDescription(lhs_dtype, shape=lhs_shape, is_encrypted=True)
+    rhs_description = ValueDescription(rhs_dtype, shape=rhs_shape, is_encrypted=True)
 
-    sample = helpers.generate_sample(parameters)
-    helpers.check_execution(circuit, function, sample, retries=3)
+    print()
```

Comment:
To cleanup

Reply:
It's meant to stay as it's super useful if something goes wrong.